server_admin/topics/users/attributes.adoc

```diff
@@ -1,11 +1,18 @@
+[configuring-user-attributes]
+= Configuring user attributes
 
-=== User Attributes
+In addition to adding basic user metadata, such as names and email, you can also store arbitrary user attributes.  
 
-Beyond basic user metadata like name and email, you can store arbitrary user attributes.  Choose a user to manage
-then click on the `Attributes` tab.
+.Prerequisite
+* You are logged into the correct realm.
```

Comment:
Is this a part of a larger workflow? If so we need it indicated somewhere. If not, I am guessing being in the "correct" realm is not the only prereq.

Reply:
@andymunro Can you help me better understand what is actually needed? I understand that the user should not be in the master realm, but what are the specific actions that the user should have engaged in before they start this procedure...what should they know?